dear_petition/petition/etl/tests/test_petition_offenses.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from dear_petition.petition.etl.load import link_offense_records_and_attachments
+from dear_petition.petition.types import dismissed
+from dear_petition.petition import constants
+from dear_petition.petition.tests.factories import (
+    CIPRSRecordFactory,
+    OffenseFactory,
+    OffenseRecordFactory,
+)
+
+pytestmark = pytest.mark.django_db
+
+
+def many_offense_records(batch, size):
+    for i in range(size):
+        record = CIPRSRecordFactory(
+            batch=batch, jurisdiction=constants.DISTRICT_COURT, county="DURHAM"
+        )
+        offense = OffenseFactory(
+            disposition_method=dismissed.DISMISSED_DISPOSITION_METHODS[0],
+            ciprs_record=record,
+            jurisdiction=constants.DISTRICT_COURT,
+        )
+        OffenseRecordFactory(offense=offense, action="CHARGED")
```

Comment:
Can you add a test where `initial_page_size == 0` and another test where `attachment_page_size == 0`?